src/js/components/Grommet/__tests__/Grommet-test.js

```diff
@@ -129,4 +130,36 @@ describe('Grommet', () => {
       });
     });
   });
+
+  test('messages', () => {
+    const component = renderer.create(
+      <Grommet messages={{
+          messages: {
+            test: {
+              label: 'My Label',
+            },
+          },
+        }}
+      >
+        <MessageContext.Consumer>
+          {({format}) => format({id: 'test.label'})}
+        </MessageContext.Consumer>
+      </Grommet>,
+    );
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+
+  test('message format function', () => {
+    const messages = {
+      "test.label": "My Label",
+    };
+    const component = renderer.create(
```

Comment:
Can we try to stay away from using `renderer.create` when creating new tests? Please see more details on https://github.com/grommet/grommet/issues/5197

Reply:
Fixed